src/lib/graph.js

```diff
@@ -59,11 +67,11 @@ const drawLevel = ({ fillLast, isMean, level, values }) => {
         return `${padding}${isLastValue && fillLast ? BAR_BODY_FILLED : unfilledBody}${padding}`
       }
 
-      if (emptyLevels === level) {
+      if (emptyLevels === level && value !== 0) {
         return `${padding}${BAR_TOP}${padding}`
       }
 
-      if (emptyLevels - 1 === level) {
+      if ((emptyLevels - 1 === level && value !== 0) || (emptyLevels === level && value === 0)) {
```

Comment:
When the value is 0 we can draw the label on the lowest level, because we do not have a `BAR_TOP` in this case.